util/lib.py

```
@@ -342,41 +343,70 @@ def load_yaml(filepath):
+
+    # Output
+    callstring.extend(["merge", "--output", o_out_path])
     logging.info(callstring)
-    
+
     try:
         check_call(callstring)
-    except Exception as e:
-        raise Exception(f"Preparing {o_path} for dashboard failed...", e)
+    except subprocess.CalledProcessError as e:
+        logging.error("Preparing %s for dashboard failed: %s", o_path, str(e))
```

Ruff

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)